sonic-thermalctld/scripts/thermalctld

```diff
@@ -643,13 +643,16 @@ class ThermalControlDaemon(DaemonBase):
         except Exception as e:
             logger.log_error('Caught exception while initializing thermal manager - {}'.format(e))
 
-        while not self.stop_event.wait(ThermalControlDaemon.INTERVAL):
+        while 1:
```

Comment:
while true?

Reply:
I heard that while 1 is faster than while true in python

Reply:
I dont think we can benefit much from "1" being faster than "true", we should give weight to readibility.

Reply:
Yes, "while 1" is not much faster than "while True" in this case. Since we have driver set full fan speed at the beginning, I suppose we can close this PR.